src/Bicep.LangServer.IntegrationTests/LangServerScenarioTests.cs

```diff
@@ -59,10 +59,15 @@ public async Task Test_Issue1931()
     }
 
     [TestMethod]
-    public async Task Test_Issue13254()
+    public async Task Test_Issue13254() // https://github.com/Azure/bicep/issues/13254
     {
+        // This test exercises the following scenario:
+        // * The user authors a file that references a module sourced from a registry
+        // * The module is re-published with different contents. The module cache (on disk) is not aware of this change
+        // * The user forces a module restore to fetch the latest contents
+
         var clientFactory = RegistryHelper.CreateMockRegistryClient("mockregistry.io", "test/foo");
-        async Task publish(string source)
+        async Task setPublishedModuleContents(string source)
```

Comment:
This is less descriptive than just calling it "publish". What does "set" & "ModuleContents" actually mean? This function **is** publishing a module.

Reply:
reverting to `publish`